src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx

```diff
@@ -62,7 +60,7 @@ const ProteinImage = (props: ProteinImageProps) => {
                 className={styles.protein}
                 y="8"
                 height={PROTEIN_HEIGHT}
-                width={scale(props.product.length as number)}
+                width={scale(props.productLength as number)}
```

Comment:
`as number` is redundant; it's defined as such in props

Reply:
Done